Combat2.0 chunk4other alex l_ merge to superbranch

1sourcecontrol opened the pull request:
Added a lot of requirements: I'm primary stat-gateing a lot of feats so it's not one-character-size fits all.
Myriad of other changes. Some are listed below.
I need a review of the changes to check for balance and aesthetics. Can someone stick lovelies on this? I can't seem to add him as a reviewer. 

Moved empty the mag to 'active feats'. Made it only apply to guns with semi-auto, so that machine-gun players won't have the option to use it since it's probably worse than their base fire rate/doesn't make sense for them.

Enhanced Neural Interface didn't seem useful as a feat, tried to buff.

Improvise Weapon, note here, i'm trying remove unarmed combat skill from the game in favor of a 'melee combat skill' to cover melee as a whole. Unarmed is nigh worthless unless you are a monk; and especially worthless if you have a sword. I believe it's beneficial to the game to group it together into a larger-covering skill, so that a typical melee character won't have to invest tons in something that _probably_ doesn't pay out 90% of the time. 
Maybe leaving unarmed in as a skill that gives benefits to grappling and unarmed hit chance (and maybe even unarmed damage) will be good for characters who want extra melee prowess. But making 'melee combat skill' more of a global control for melee would make melee more accessible--and more likely to happen as a whole.

Run n' gun, added a stamina cost, moved to active feats

Buffed salty hashtables. I believe the spirit of the rules should be to balance the feats and make them similar in power to one-another. Globally granting a +20% bonus to w/e value the feat adjusts, will certainly not be balanced in many cases.

Buffed steady shot, added stamina cost, and moved to active feats.

1sourcecontrol committed:
started going through feats for c2.0

Also, Concussive maintenance description wasn't clear so i tried to reword it.

1sourcecontrol committed:
more feat updates

1sourcecontrol committed:
c2.0 feats

moved empty the mag to 'active feats'. Made it only apply to guns with semi-auto, so that machine-gun players won't have the option to use it since it's probably worse than their base fire rate/doesn't make sense for them.

Enhanced Neural Interface didn't seem useful as a feat, tried to buff.

Improvise Weapon, note here, i'm trying remove unarmed combat skill from the game in favor of a 'melee combat skill' to cover melee as a whole. Unarmed is nigh worthless unless you are a monk; and especially worthless if you have a sword. I believe it's beneficial to the game to group it together into a larger-covering skill, since a typical melee character won't have to invest tons to be really useful in something that _probably_ doesn't pay off 90% of the time. 
Maybe leaving unarmed in as a skill that gives benefits to grappling and unarmed hit chance (and maybe even unarmed damage) will be good for characters who want extra melee prowess. But making 'melee combat skill' more of a global control for melee would make melee more accessible--and more likely to happen as a whole.

run n' gun, added a stamina cost, moved to active feats

Buffed salty hashtables. I believe the spirit of the rules should be to balance the feats and make them similar in power to one-another. Globally granting a +20% bonus to w/e value the feat adjusts, will certainly not be balanced in many cases.

Buffed steady shot, added stamina cost, and moved to active feats.

Added a lot of requirements: primary stat-gates a lot of feats so it's not one-size fits all.

Myriad of other changes.

1sourcecontrol committed:
moar feat balancing

1sourcecontrol committed:
last of the feats, except rainman

1sourcecontrol committed:
removed nanites from feat cost

additional work on balancing the feats

1sourcecontrol commented:
Moved all the feat active costs to nanites.

FYI, I'm intending stamina to regenerate some every round (maybe 5 per turn) and reducing the amount of stamina that players get compared to nanites. Player Class's Primary Attribute * 10 = stamina pool.
So that explains why stamina costs are roughly 2/3rds of nanite costs, or otherwise small costs

1sourcecontrol committed:
more clean-up, balancing

1sourcecontrol committed:
minor bugfixes

1sourcecontrol committed:
removed nanites from the basic rules, changed some names for shields and cloaking stuff

1sourcecontrol committed:
changed melee to allow doing some damage (blunt) through armor

1sourcecontrol committed:
typo

1sourcecontrol committed:
updated melee blunt damage

1sourcecontrol committed:
minor fixes

1sourcecontrol committed:
updated cloaking rules

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -42,227 +42,211 @@ Acquired: . . . Who Needs Nano-Technologies?
-basic computer inquiry or operation. +10% to ONE pertinent class skill 
-(hacking, etc.) while using this feat, to be chosen at the time of taking 
-this feat.
+Prerequisites: INT 3, You must be a Cyborg or have a way to interface your mind to electronics
```
Should be available for all races

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -42,227 +42,211 @@ Acquired: . . . Who Needs Nano-Technologies?
+Allows the user to better concentrate on their expenditure of nanites, allowing them
+to achieve one of the following (you must choose one option when you acquire the feat):
+    +2 turns of cloaking
+    Once per turn, +20% (or +2) to a magic-combat-action at a cost of 20 nanites (ask GM before applying to an action.)
```
Nanite related, not magic.

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -42,227 +42,211 @@ Acquired: . . . Who Needs Nano-Technologies?
 holding something, you can drop it and still draw your weapon as a free 
 action. However, you might not want to drop anything glass. Or any explosives.
 
 Rainman: 
```
Buff possibly required?

1sourcecontrol replied:
@trowl223 
I made it a req for "Blur". Maybe that's enough? 
Quickdraw helps weapons that don't have access to one-point slings, as well.

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -271,45 +255,34 @@ upgraded by making good decisions based on this skill, rather than by
 allowing you to process everything around you more quickly for a split second. 
-When a target is within half your effective range, your hit DC is decreased 
-by 10%.
+When a target is within half your largest range bracket, your miss chance is 
```
Target within R1 or R2

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -466,62 +452,101 @@ Proficient: Explosives
+	Knowledge [Nanites] > 40
+Add +3 miss chance to a piece of cover for 4 turns
+
+Combat Roll [30 Stamina]:
```
Add that the miss chance applies to reflex rolls made by enemies too, example being pinned by a sniper or LMG

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -466,62 +452,101 @@ Proficient: Explosives
+cover. You hunker or go prone behind that cover as a free action. Cannot 
+be used in concert with Run 'N' Gun.
+
+Empty the Mag: 
```
Should be moved to passive feats

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -466,62 +452,101 @@ Proficient: Explosives
+attack still apply.
 
-Gun Kata [30 Nanites / Turn]: 
+Gun Kata [20 Stamina]: 
```
Increase cost to 30 or 40?

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -466,62 +452,101 @@ Proficient: Explosives
+increasing the DC by 20 per strike instead of increasing the strike's miss chance.
 
-Multi-Target [ X*40 Nanites]: 
+Multi-Target [13*X Stamina]: 
```
Chance to 4 / 2 actions?

1sourcecontrol replied:
@trowl223  I think it would be too weak compared to just a gun-kata. Also, i just realized we didn't add a clause for what 'roll' people make to see if you succeed with multi-target. So I did roll to hit 'your first target'? Which means players will always choose the easiest one..

trowl223 commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -466,62 +452,101 @@ Proficient: Explosives
 
-Twitch	[10 Nanites]:
-Prerequisites: Level 3, Trained in SMGs or Pistols, Dex > 6
+Run 'N' Gun [10 Stamina]: 
```
Add comment about aiming while using this

1sourcecontrol committed:
reverted back to nanites from stamina

1sourcecontrol committed:
changed stamina back to nanites, implemented other bugfixes

1sourcecontrol commented:
@trowl223, FYI, I decided to buff weapon familiarization to 1.5 miss chance reduction, like you were musing about. it seemed a little below other abilities. 

In basic rules doc, I still need to define Nanites as depending on a class's primary attribute (or whatever we want to call it). Change it to 10* primary attribute, instead of 10*x +50. And add some rule about regeneration during battle.

1sourcecontrol committed:
reverted a feat's name

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -356,6 +358,7 @@ calculations, so if you had 10 int, and increased to 11, you would get 22 skill
 for the level. Additionally, any stat point alteration forces a recalculation on all
 affected values.
 
+			  LEVELING-UP TABLE
```
I <3 chart titles! Looks much cleaner!

bleehu commented on play/PlayerRules/Basic Rules.txt:
```diff
@@ -855,7 +858,11 @@ Explosives:
+	you then deal a special kind of concussive damage (called blunt damage) that ignores armor but you can 
+	only deal 1/2 damage, up to a max of 25 damage. Max of 35 damage if you are using a melee weapon (like a
+	club, sword, or even hitting them with your gun).
 Armor:
```
I really like this rule.

1sourcecontrol replied:
I'm not sure we need this rule now, now that we have the ability to ignore armor with a sufficiently accurate melee attack

bleehu replied:
I really like that it defines blunt damage. Let's leave it in! Unless you really want to clean it out.

bleehu commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -42,227 +42,226 @@ Acquired: . . . Who Needs Nano-Technologies?
-Roll a d10 before any Blind Firing or Hip Firing action. If the roll is from 6 to 10, 
-Gain -2 miss chance for that firing action; If the roll is from 1 to 5, nothing happens.
+Prerequisites: Level 3 and Assassin or Monk.
+Any damage dealt to an unaware opponent from behind deals an additional +20 damage.
```
Can we make this damage scale with level? Or with a primary stat?

bleehu commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -42,227 +42,226 @@ Acquired: . . . Who Needs Nano-Technologies?
+	*If you have a crit range greater than or equal to 3, only rolls of 8 or more will
+trigger Double Tap. One exception: the max crit range of 3 can be increased to 4 by the 
+feat "Never Tell Me the Odds". If you have a crit range of less than 3, then crit range applies
+as normal. 
```
I like this! I especially like that we're merging things with anti-Lovelies-esque breaking patches thought out beforehand!

bleehu commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -42,227 +42,226 @@ Acquired: . . . Who Needs Nano-Technologies?
-Prerequisites:
-When clearing a room, add 10 to your reflex roll.
+Prerequisites: PER 6
+When clearing a room, add 20 to your reflex roll.
```
Let's add some flavor text to this one. Doesn't have to be much, but let's keep it consistent.

bleehu commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -346,29 +335,38 @@ Enables the use of SMGs by the Engineer class.
+Prerequisites: Level 5, LUK 5, INT 5 
+You like to do a little investing in your free time. This feat can only
+be taken once. Investor can be used twice, but only once every 5 levels.
+(Roll a d10 + Luck level) x 125. Gain this value in credits.
```
Much improved.

bleehu commented on play/PlayerRules/CharacterCreation/03_Feats.txt:
```diff
@@ -42,227 +42,226 @@ Acquired: . . . Who Needs Nano-Technologies?
+	Rainman: 
+	Prerequisites: CHA <3 and INT >8
+	Your character has something going for them. You may ask the GM how many 
+	bullets each opponent has left in their weapon.
```
I'm not coming up with anything good that makes sense. Let's yank it until we can come up with something better. 
I'll do this in a separate PR.

bleehu reviewed (changes requested):
All in all, a fantastic update! Just a minor tweak or two. See comments. Once these are addressed to your satisfaction, Shipit!

1sourcecontrol committed:
fixes for mike's review

extra text fix too

1sourcecontrol committed:
defined nanites in basic rules w/ other

defined how nanites are gained in the basic rules with a class's primary attribute

1sourcecontrol committed:
formatting/text fixes

1sourcecontrol committed:
Merge branch 'combat2.0superbranch_AlexL' into combat2.0_chunk4other_AlexL

1sourcecontrol committed:
fixed cloaking text contradiction

1sourcecontrol committed:
melee armor rules

added rules to allow players to ignore armor with a sufficiently accurate hit, during a melee attack.
also re-arranged the layout of ranged/melee rules. Added in some notes to allude to the fact that
melee players will have modifiers from their attack/weapon

1sourcecontrol committed:
modified melee rules for d10 scale

also modified for always using guard DC

1sourcecontrol committed:
monk/gunslinger minor update

1sourcecontrol committed:
minor gun-kata rebalance

reduced miss chance from 5 to 1

1sourcecontrol committed:
rebalance flash dance, gun kata, and prereq multi-target

1sourcecontrol committed:
added a level 10 bonus to stealth attacks

1sourcecontrol committed:
adjusted paladin nanite costs and some defensive bonuses for c2.0

1sourcecontrol committed:
updated damages to reflex there "base damage" or bonus" terminology.. 

other minor changes

1sourcecontrol committed:
improved stealth attack rules

1sourcecontrol committed:
armor table of contents

1sourcecontrol committed:
minor fix to melee text

1sourcecontrol committed:
added a range limit rule for un-magnified shooting

1sourcecontrol committed:
ranged weapon technicality

bleehu reviewed (approved):
shipit baby!

1sourcecontrol committed:
restructured skills and put in 2 max skill limits
